tests/EvmAcceptanceTests/helper/GeneralHelper.js

```diff
@@ -18,6 +18,10 @@ var general_helper = {
     return hre.network.config.url;
   },
 
+  getWebsocketUrl: function () {
+    return hre.network.config.websocketUrl;
```

Comment:
where did you get it? I mean `websocketUrl` is it a custom field? if so, you should add it to hardhat.config.js as well and skip these tests for non-zilliqa networks